packages/error-reporting/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@google-cloud/error-reporting",
+  "version": "0.1.0",
+  "description": "Stackdriver Error Reporting Client Library for Node.js",
+  "main": "index.js",
+  "repository": "GoogleCloudPlatform/google-cloud-node",
+  "scripts": {
+    "test": "$(npm bin)/nyc --exclude=\"fuzzer.js\" $(npm bin)/mocha ./test/unit/*.js",
+    "integration-tests": "$(npm bin)/nyc --exclude=\"error-message.js\" $(npm bin)/mocha ./test/system-test/*.js",
+    "style": "$(npm bin)/jshint lib index.js",
```

Comment:
`lint` for the task name, if you want to be consistent with the repo.

There is no `lib`.